FAQ/FAQ-Wallet.md

```diff
@@ -15,6 +16,13 @@ BTCPay has an internal wallet which you can use to see the transactions and even
 
 You'll have to confirm and sign the transaction on your hardware wallet. BTCPay wallet is not a hot wallet. The private keys are secured inside your hardware wallet.
 
+## What is wallet re-scan in BTCPay?
+
+A feature called [scantxoutset](https://bitcoincore.org/en/doc/0.17.0/rpc/blockchain/scantxoutset/) added in Bitcoin Core 0.17.0 enables BTCPay users to easily restore an old wallet in BTCPay and see a proper ballance, without ever having to use an external wallet.
+Prior to version 1.0.3.0 which added this feature, if a user added an xpub key already used for receiving payments, BTCPay couldn't recognize those past UTXOs, and would start generating addresses for invoices that were already used in the past.
+
+Wallet re-scan solves allows users to easier import their xpub keys that was used in the past, for example on a mobile wallet or even on another BTCPay Server. That is why, users who import xpub that was used for receiving payments in the past, should re-scan their wallet. Additionally this feature helps wit gap limit issues explained later in this document.
+
```

Comment:
I think this is difficult to understand. It would be better explained with a real example like:

1. You use a wallet outside of BTCPay (for example Ledger Nano S with the Ledger Live app)
2. You receive some transactions on it
3. You decide to use this wallet with a new BTCPay Store (Add derivation scheme...)
4. You then find out that BTCPay has no information about past transactions, and show the new wallet's balance at 0

This is because when you use a new derivation scheme with your store, BTCPayServer has no idea about the past transactions of your wallet.
To solve this issue, you need to use the Rescan feature. The Rescan feature relies on Bitcoin Core 0.17.0's  [scantxoutset](https://bitcoincore.org/en/doc/0.17.0/rpc/blockchain/scantxoutset/) to scan the current state of the blockchain (called UTXO Set) for coins belonging to your derivation scheme.
Once the scan is complete, BTCPay Server will show the correct balance of your wallet, along with some of past transactions of your wallet.

Reply:
Perfect example, will do that later, thanks.